echopype/visualize/api.py

```diff
@@ -206,6 +206,11 @@ def _add_water_level(
         return range_in_meter
     if isinstance(water_level, xr.DataArray):
         check_dims = range_in_meter.dims
+
+        # rename time3 to ping_time because range_in_meter is in ping_time
```

Comment:
How about moving this statement to the start of the function? That way it'll also encompass the similar renaming in line 199